src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,6 +5,16 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
```

Comment:
Don't begin class or method implementation with empty lines (lines 3-6). Remove these to comply with the code style.